packages/forma-36-react-components/src/components/List/List.tsx

```diff
@@ -3,25 +3,33 @@ import cn from 'classnames';
 
 import styles from './List.css';
 
+type ListType = 'ul' | 'ol';
+
 export interface ListProps {
+  as?: ListType;
   className?: string;
   children: React.ReactNode;
   style?: React.CSSProperties;
   testId?: string;
 }
 
 export function List({
+  as: Tag = 'ul',
```

Comment:
@nikazawila maybe we could follow one of the existing options for those kinds of props? We have `element` in couple of components or `htmlTag` in `<Flex />` and `<Grid />`. We should probably have just one ( a bit complicated to migrate I guess). But maybe we could avoid introducing another pattern to this family 😉 WDYT?

Reply:
sure, consistency is king. I guess I did poor research on how it is handled atm. Two ways are already plenty 😉
Will adjust.

Reply:
Awesome! Thank you 😊